src/main/java/org/terasology/launcher/settings/LauncherSettings.java

```diff
@@ -86,8 +89,12 @@ public synchronized void init() {
 
     public abstract String getUserJavaParameters();
 
+    public abstract String getBaseJavaParameters();
+    
     public synchronized List<String> getUserJavaParameterList() {
```

Comment:
Should this be renamed to `getJavaParameterList()`? It now contains both base params and user params.